sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/InstancesClient.cs

```diff
@@ -0,0 +1,846 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// Time Series Insights instances client.
+    /// </summary>
+    public class InstancesClient
```

Comment:
What is the guideline on naming these? should we call them `Instances` or `InstancesClient`?

Reply:
ah, I see what you are doing there, I like it. The property is called `Instances` but its type is `InstancesClient`